constraint/go.mod

```diff
@@ -3,15 +3,26 @@ module github.com/open-policy-agent/frameworks/constraint
 go 1.13
 
 require (
+	github.com/OneOfOne/xxhash v1.2.8 // indirect
```

Comment:
Should this PR just focus on updating `github.com/open-policy-agent/opa v0.24.0`?

Reply:
I might have done something wrong, but that was my intention. I believe this is pulled in as a transitive dependency of OPA, but I can double check in the morning.

Reply:
https://github.com/open-policy-agent/opa/blob/v0.24.0/go.mod#L6 👈 yeah I think this is correct. It's a dependency of OPA, so upgrading OPA would pull this in.